README.md

```diff
@@ -63,6 +63,9 @@ Loads all modules `require()`d in the provided Lua string, and returns the resul
 | **postprocess** | `(module: Module, options: RealizedOptions) => string` | `undefined` | Postprocess a module, immediately before its added to the bundle.  |
 | **preprocess** | `(module: Module, options: RealizedOptions) => string` | `undefined` | Preprocess a module, before luabundle makes any of its own modifications.  |
 | **rootModuleName** | `string` | `"__root"` | The contents of `inputFilePath` are interpreted as module with this name.  |
+| **builtModules** | `string[]` | `[]` | List of builtin modules that runtime will provide.  |
```

Comment:
`builtModules` implies this is only useful for system provide modules, but it could be used for other reason too. Could you please rename this to `ignoredModuleNames`?